brew.sh

```diff
@@ -26,6 +26,12 @@ brew install bash
 brew tap homebrew/versions
 brew install bash-completion2
 
+# Switch to using brew-installed bash as default shell
+if ! fgrep -q '/usr/local/bin/bash' /etc/shells; then
+  echo '/usr/local/bin/bash' >> /etc/shells;
```

Comment:
I knew there was a reason for `tee`, but I blindly followed the suggestion on the other PR. That'll teach me: trust, but verify. 

I'm AFK for a couple of days, I'll get this back to its previous incarnation & push that up to get this working again.